.gitmodules

```diff
@@ -0,0 +1,7 @@
+[submodule "test/kmtest"]
```

Comment:
Let's change submodules to cmake fetch. It will be easier to manage them and git won't download them if tests are not required.

Reply:
made it